server/db/sequelize/models/users.js

```diff
@@ -0,0 +1,95 @@
+import Promise from 'bluebird';
+import bcryptNode from 'bcrypt-nodejs';
+const bcrypt = Promise.promisifyAll(bcryptNode);
```

Comment:
For this I was mostly concerned about not handling all the cases properly, where I know bluebird is safe. I'm not sure es6-promise has a promisify method, but if you look at [bluebird's promise anti-patterns](https://github.com/petkaantonov/bluebird/wiki/Promise-anti-patterns) they mention:

> The generic promisification is likely to be faster because it can use internals directly but also handles edge cases like libraryFunction throwing synchronously or using multiple success values.

I know I could handle these cases myself but there could be more I'm not aware of on top of this.